bin/client/main.rs

```diff
@@ -108,6 +109,50 @@ async fn main() -> std::result::Result<(), ClientError> {
             .await?;
             println!("{}", toml::to_string_pretty(&label)?);
         }
+        SubCommand::CreateKey(create_opts) => {
+            let dir = create_opts.secret_file.unwrap_or_else(|| {
+                default_config_dir()
+                    .unwrap_or_else(|| PathBuf::from("."))
+                    .join("secret_keys.toml")
+            });
+            println!("Writing keys to {}", dir.to_string_lossy());
+
+            match std::fs::metadata(dir.clone()) {
+                Err(_) => {
+                    println!(
+                        "File {} does not exist. Creating it.",
+                        dir.to_string_lossy()
+                    );
+                    let mut keyfile = SecretKeyFile::default();
+                    let newkey = SecretKeyEntry::new(
+                        create_opts.label,
+                        vec![bindle::SignatureRole::Creator],
+                    );
+                    keyfile.key.push(newkey);
+                    keyfile
+                        .save_file(dir)
+                        .map_err(|e| ClientError::Other(e.to_string()))?;
+                }
+                Ok(info) => {
+                    if !info.is_file() {
+                        eprint!("Path must point to a file.");
+                        return Err(ClientError::Other(
+                            "Keyfile cannot be directory or symlink".to_owned(),
```

Comment:
Nit: I think we might want to support symlinks if they point at a regular file

Reply:
Storing a key file behind a symlink is insecure because of the ambiguities of both permissions and the underlying filesystem. (For example, a malicious user who can modify mount tables could trick the client into loading a different key or otherwise modify the file at the other end of the symlink). The recommendation is to never reference private keys via symlink.